app/models/course/level.rb

```diff
@@ -0,0 +1,4 @@
+class Course::Level < ActiveRecord::Base
+  belongs_to :course, inverse_of: :levels
+  validates :exp_threshold, numericality: { greater_than: 0 }
```

Comment:
@lowjoel do you mean that for exp_threshold? I think it is the appropriate term already?